### DISCUSSION TITLE
How I connet to the Windows ComfyUI server from Mac Krita plugin

### DISCUSSION THREAD

**Discussion by delizade** [Acly/krita-ai-diffusion#1058] [Q&A] [answered]
Hi all,
I installed comfyUI for the plugin and it is working on windows machine. But I need to assign a local server IP for the mac side but I couldn't.
I disabled firewall and windows defender. then I changed Settings.json file to change local IP for the server like 192.168.0.15:8188
I can connect to the windows machine with that IP from Mac but I can't connect to the port I guess.

Do you have any idea what the problem might be?

thank you

**Comment by Hypersonic-Popcorn** [comment, marked as answer]
You need to add the --listen argument when you launch ComfyUI. Make sure that you're doing this on a local network that you can trust as anyone would be able to connect to your Windows box and generate images or maybe more.

**Reply by rbfbatx** [reply]
How do I add the listen argument?
